Shared/TLS/Get-TlsCipherSuiteInformation.ps1

```diff
@@ -12,6 +12,37 @@ function Get-TlsCipherSuiteInformation {
     )
 
     begin {
+
+        function GetProtocolNames {
+            param(
+                [int[]]$Protocol
+            )
+            $protocolNames = New-Object System.Collections.Generic.List[string]
+
+            foreach ($p in $Protocol) {
+                $name = [string]::Empty
+
+                if ($p -eq 2) { $name = "SSL_2_0" }
```

Comment:
Do we have a reference for this mapping that we can add as a comment?

Reply:
Not a Microsoft source. 

https://github.com/tintinweb/scapy-ssl_tls/blob/8fde2f50d0849f97ae7916d9e985a6595fe0999f/scapy_ssl_tls/ssl_tls.py#L249

Reply:
Hm yeah, looks like this is specified within the RFC per version. A good overview can also be found here: https://tlsfingerprint.io/top/versions 
I think we should be good then.